rs_filter/filter_base.rs

```diff
@@ -246,12 +225,28 @@ impl RootContext for HttpHeadersRoot {
 }
 
 pub struct HttpHeaders {
-    context_id: u32,
-    workload_name: String,
-    target_graph: Graph<(String, IndexMap<String, String>), String>,
+    pub context_id: u32,
+    pub workload_name: String,
+    pub target_graph: Graph<(String, IndexMap<String, String>), String>,
 }
 
-impl Context for HttpHeaders {}
+impl Context for HttpHeaders {
+    fn on_http_call_response(
+        &mut self,
```

Comment:
How do you know that this is from storage?  Storage should never be sending a filter anything, right?

Reply:
As far as I understand this is a callback that handles the return from the asynchronous `dispatch_http_call`. We are only doing this when we are sending to storage. But it does not quite work yet.

Reply:
Could you put a comment saying it doesn't work then?  I think it is useful to distinguish between the stuff we have working and the stuff we're experimenting with.